src/prerender.ts

```diff
@@ -213,7 +215,11 @@ export async function prerender(nitro: Nitro) {
       );
       for (const _link of extractedLinks) {
         if (canPrerender(_link)) {
-          routes.add(_link);
+          if (_link.endsWith("_payload.json")) {
```

Comment:
sorry but i cannot let this to be merged in nitro. please consider proposing framework agnostic solutions AND make sure there is a nitro minimal reproduction that makes sure it actually proves and solves an issue instead of hiding root causes.